include/deal.II/non_matching/mapping_info.h

```diff
@@ -799,7 +811,8 @@ namespace NonMatching
   MappingInfo<dim, spacedim, Number>::reinit_surface(
     const IteratorRange<Iterator> &                    cell_iterator_range,
     const std::vector<ImmersedSurfaceQuadrature<dim>> &quadrature_vector,
-    const unsigned int                                 n_unfiltered_cells)
+    const unsigned int                                 n_unfiltered_cells,
```

Comment:
Are you using `reinit_surface`? It seems not be tested?

Reply:
I guess you don't fill a `ImmersedSurfaceQuadrature` in your code? Since, `QSimplex::compute_affine_transformation()` does not return such a type.

Reply:
I only use reinit_faces() and reinit_cells(). So I can get rid of the flag in reinit_surfaces(). Or, since we seem to agree on using AD: add an assert to ensure reinit_surfaces() cannot be used with global weights.

Reply:
Let's do this :+1: